pkgs/applications/misc/rootbar/default.nix

```diff
@@ -41,5 +41,6 @@ stdenv.mkDerivation rec {
     license = licenses.gpl3Plus;
     maintainers = with maintainers; [ AndersonTorres ];
     platforms = platforms.unix;
+    broken = stdenv.isDarwin;
```

Comment:
```suggestion
    platforms = platforms.linux;
```

Reply:
Not quite, because of https://docs.freebsd.org/en/books/handbook/wayland/ and probably others.

Reply:
Support for Unix platforms is not automatic: they require at least explicit `epoll-shim` support from the package, or one will only encounter errors even when Wayland is available on the platform.